basenko_denis/src/components/MessageField/MessageField.jsx

```diff
@@ -0,0 +1,76 @@
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import './messagesField.css'
+
+import React from 'react';
+
+import { MessagesList } from 'components/MessagesList/MessagesList';
+
+export class MessageField extends React.Component {
+    
+    state ={
+        messages:[
+            {author:'bot',text:'Привет я бот Валера'},
+            {author:'bot',text:'Я отвечу на любой твой вопрос.'},
+            {author:'bot',text:'Задавай?'},
+        ],
+        newMessage: '',
+        author:''
```

Comment:
Рекомендую оставлять trailing comma. Напомните на уроке - расскажу зачем :)
```suggestion
        author: '',
```